app/code/Magento/Review/Ui/DataProvider/Product/ReviewDataProvider.php

```diff
@@ -79,7 +87,8 @@ public function getData()
     }
 
     /**
-     * {@inheritdoc}
+     * @inheritdoc
+     * @return mixed
```

Comment:
As far as I can see it doesn't return anything. Am I missing something? This change should fix the Static Tests and Semantic Version Checker.

```suggestion
     * @return void
```

Reply:
@eduard13  thanks for suggestion

Reply:
@eduard13  check the screenshot it shows error  http://prntscr.com/uz4ukj
but i don't change in it. this function has as it is.